asllib/doc/ASLTypingReference.tex

```diff
@@ -6810,17 +7660,15 @@ \chapter{Typing of Specifications}
 An ASL specification consists of a list of declarations.
 Type checking the specification is considered successful if all declarations can be successfully annotated.
 
-\newcommand\decls[0]{\texttt{decls}}
-
 \section{TypingRule.Specification \label{sec:TypingRule.Specification}}
 \subsection{Prose}
-Annotating an ASL specification \texttt{decls} in an environment \texttt{env} results in a rewritten specification \texttt{decls'}
-and a new environment \texttt{new\_env} and all of the following apply:
+Annotating an ASL specification $\decls$ in an environment $\tenv$ results in a rewritten specification $\declsp$
```

Comment:
rewritten, in what sense?

Reply:
annotated

Reply:
thank you - have you opted to use the term "annotated" then?

Reply:
yes